entry_timeseries.go

```diff
@@ -83,6 +91,33 @@ func (entry TimeseriesEntry) Create() error {
 	return makeErrorOrNil(err)
 }
 
+// InsertColumns : insert columns in a existing timeseries
+func (entry *TimeseriesEntry) InsertColumns(cols ...TsColumnInfo) error {
+	colsToInsert := TsColumnInfos(cols)
+	alias := C.CString(entry.alias)
+	columnsCount := C.qdb_size_t(len(colsToInsert))
+	columnsArray := colsToInsert.toStructC()
+	var columns *C.qdb_ts_column_info_t
+	if columnsCount != 0 {
```

Comment:
Why do you check for that? You may leave the C API handle the validation of user input, hmm?